### DISCUSSION TITLE
Autobrr trigger plex scan (willing to tip)

### DISCUSSION THREAD

**Discussion by zazabozaza** [autobrr/autobrr#1445] [Q&A]
Willing to tip 10$ to whoever finds a workaround. Basically, I want Autobrr to trigger a plex library scan. My setup currently has [cloudboxes autoscan](https://github.com/Cloudbox/autoscan) installed which uses webhooks and I was wondering if I can use that to trigger a Plex library scan, not a full scan want it to be just for 1 library.  

It can be as simple as having autobrr enter a url because you can trigger a scan by url commands for example
http://[PMS_IP_ADDRESS]:32400/library/sections/29/refresh?X-Plex-Token=YourTokenGoesHere this url

**Comment by KyleSanderson** [comment]
Webhook.

**Comment by zze0s** [comment]
Depending on your download client it might be best way to do an On completion script with a simple bash script that has a curl command and path if needed/useful. It's supported with qbit and deluge, probably more clients.
